tests/Annotations/Serializer/AnnotationNormalizerTest.php

```diff
@@ -0,0 +1,689 @@
+                            'text' => 'Final paragraph.',
+                        ],
+                    ],
+                    'created' => $createdDate,
+                    'document' => [
+                        'title' => 'title',
+                        'uri' => 'uri',
+                    ],
+                    'parents' => [],
+                ],
+                new Annotation(
+                    'id',
+                    "List:\n\n- Item 1\n- Item 2  \n\n1. Item 1\n1. Item 2\n1. Item 3\n\nFinal paragraph.",
+                    new DateTimeImmutable($createdDate),
+                    new DateTimeImmutable($createdDate),
+                    new Annotation\Document('title'),
+                    new Annotation\Target('source'),
+                    'uri',
+                    null,
+                    new Annotation\Permissions(Annotation::PUBLIC_GROUP)
+                ),
+            ],
+            'markdown-nested-lists' => [
+                [
+                    'id' => 'id',
+                    'access' => 'public',
+                    'content' => [
+                        [
+                            'type' => 'paragraph',
+                            'text' => 'Nested list:',
+                        ],
+                        [
+                            'type' => 'list',
+                            'prefix' => 'bullet',
+                            'items' => [
+                                'Item 1',
+                                'Item 2',
+                                [
+                                    [
+                                        'type' => 'list',
+                                        'prefix' => 'bullet',
+                                        'items' => [
+                                            'Item 2.1',
+                                            [
+                                                [
+                                                    'type' => 'list',
+                                                    'prefix' => 'number',
+                                                    'items' => [
+                                                        'Item 2.1.1',
+                                                    ],
+                                                ],
+                                            ],
+                                        ],
+                                    ],
+                                ],
+                            ],
+                        ],
+                    ],
+                    'created' => $createdDate,
+                    'document' => [
+                        'title' => 'title',
+                        'uri' => 'uri',
+                    ],
+                    'parents' => [],
+                ],
+                new Annotation(
+                    'id',
+                    "Nested list:\n\n- Item 1\n- Item 2\n  - Item 2.1\n    1. Item 2.1.1",
```

Comment:
wonder if there's any format that would make it easier to provide many tests like these. Separating the data into text files may introduce too much indirection, but the `\n` impact readability

Reply:
Heredoc is probably uglier. What about:
```
                    "Nested list:\n". 
                    "\n". 
                    "- Item 1\n". 
                    "- Item 2\n".
                    "  - Item 2.1\n".
                    "    1. Item 2.1.1" 
```
or even
```
                    $this->lines([
                        "Nested list:", 
                        "", 
                        "- Item 1", 
                        "- Item 2",
                        "  - Item 2.1\n",
                        "    1. Item 2.1.1",
                    ]) 
```
?